### ISSUE TITLE
Issues after typing npm install command

### ISSUE THREAD

**Issue by ysy5700** [cozie-app/cozie#111]
Microsoft Windows [Version 10.0.22000.556]
(c) Microsoft Corporation. All rights reserved.

C:\Users\윤서연\Documents\GitHub\cozie>npm install
npm WARN old lockfile
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile
npm WARN deprecated rollup-plugin-node-resolve@5.2.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-node-resolve.
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated debug@3.2.6: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
npm WARN deprecated rollup-plugin-commonjs@10.1.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-commonjs.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated core-js@2.6.11: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
npm ERR! code 1
npm ERR! path C:\Users\윤서연\Documents\GitHub\cozie\node_modules\keytar
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c prebuild-install || node-gyp rebuild
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.0.0
npm ERR! gyp info using node@17.8.0 | win32 | x64
npm ERR! gyp ERR! find Python
npm ERR! gyp ERR! find Python Python is not set from command line or npm configuration
npm ERR! gyp ERR! find Python Python is not set from environment variable PYTHON
npm ERR! gyp ERR! find Python checking if "python3" can be used
npm ERR! gyp ERR! find Python - "python3" is not in PATH or produced an error
npm ERR! gyp ERR! find Python checking if "python" can be used
npm ERR! gyp ERR! find Python - "python" is not in PATH or produced an error
npm ERR! gyp ERR! find Python checking if Python is C:\Users\윤서연\AppData\Local\Programs\Python\Python39\python.exe
npm ERR! gyp ERR! find Python - "C:\Users\윤서연\AppData\Local\Programs\Python\Python39\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files\Python39\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files\Python39\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Users\윤서연\AppData\Local\Programs\Python\Python39-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Users\윤서연\AppData\Local\Programs\Python\Python39-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files\Python39-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files\Python39-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files (x86)\Python39-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files (x86)\Python39-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Users\윤서연\AppData\Local\Programs\Python\Python38\python.exe
npm ERR! gyp ERR! find Python - "C:\Users\윤서연\AppData\Local\Programs\Python\Python38\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files\Python38\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files\Python38\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Users\윤서연\AppData\Local\Programs\Python\Python38-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Users\윤서연\AppData\Local\Programs\Python\Python38-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files\Python38-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files\Python38-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files (x86)\Python38-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files (x86)\Python38-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Users\윤서연\AppData\Local\Programs\Python\Python37\python.exe
npm ERR! gyp ERR! find Python - "C:\Users\윤서연\AppData\Local\Programs\Python\Python37\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files\Python37\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files\Python37\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Users\윤서연\AppData\Local\Programs\Python\Python37-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Users\윤서연\AppData\Local\Programs\Python\Python37-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files\Python37-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files\Python37-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files (x86)\Python37-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files (x86)\Python37-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Users\윤서연\AppData\Local\Programs\Python\Python36\python.exe
npm ERR! gyp ERR! find Python - "C:\Users\윤서연\AppData\Local\Programs\Python\Python36\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files\Python36\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files\Python36\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Users\윤서연\AppData\Local\Programs\Python\Python36-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Users\윤서연\AppData\Local\Programs\Python\Python36-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files\Python36-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files\Python36-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if Python is C:\Program Files (x86)\Python36-32\python.exe
npm ERR! gyp ERR! find Python - "C:\Program Files (x86)\Python36-32\python.exe" could not be run
npm ERR! gyp ERR! find Python checking if the py launcher can be used to find Python 3
npm ERR! gyp ERR! find Python - executable path is "C:\Users\������\AppData\Local\Programs\Python\Python310\python.exe"
npm ERR! gyp ERR! find Python - "C:\Users\������\AppData\Local\Programs\Python\Python310\python.exe" could not be run
npm ERR! gyp ERR! find Python
npm ERR! gyp ERR! find Python **********************************************************
npm ERR! gyp ERR! find Python You need to install the latest version of Python.
npm ERR! gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
npm ERR! gyp ERR! find Python you can try one of the following options:
npm ERR! gyp ERR! find Python - Use the switch --python="C:\Path\To\python.exe"
npm ERR! gyp ERR! find Python   (accepted by both node-gyp and npm)
npm ERR! gyp ERR! find Python - Set the environment variable PYTHON
npm ERR! gyp ERR! find Python - Set the npm configuration variable python:
npm ERR! gyp ERR! find Python   npm config set python "C:\Path\To\python.exe"
npm ERR! gyp ERR! find Python For more information consult the documentation at:
npm ERR! gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
npm ERR! gyp ERR! find Python **********************************************************
npm ERR! gyp ERR! find Python
npm ERR! gyp ERR! configure error
npm ERR! gyp ERR! stack Error: Could not find any Python installation to use
npm ERR! gyp ERR! stack     at PythonFinder.fail (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-python.js:330:47)
npm ERR! gyp ERR! stack     at PythonFinder.runChecks (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-python.js:159:21)
npm ERR! gyp ERR! stack     at PythonFinder.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-python.js:248:16)
npm ERR! gyp ERR! stack     at PythonFinder.execFileCallback (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-python.js:294:16)
npm ERR! gyp ERR! stack     at exithandler (node:child_process:406:5)
npm ERR! gyp ERR! stack     at ChildProcess.errorhandler (node:child_process:418:5)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:527:28)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:289:12)
npm ERR! gyp ERR! stack     at onErrorNT (node:internal/child_process:476:16)
npm ERR! gyp ERR! stack     at processTicksAndRejections (node:internal/process/task_queues:83:21)
npm ERR! gyp ERR! System Windows_NT 10.0.22000
npm ERR! gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd C:\Users\윤서연\Documents\GitHub\cozie\node_modules\keytar
npm ERR! gyp ERR! node -v v17.8.0
npm ERR! gyp ERR! node-gyp -v v9.0.0
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\윤서연\AppData\Local\npm-cache\_logs\2022-04-05T07_41_16_550Z-debug-0.log

C:\Users\윤서연\Documents\GitHub\cozie>

**Comment by mariofrei** [comment]
Hi @ysy5700,
Could you add a bit of description about what you did to run into these errors? e.g., at what step of the tutorial did you encounter this issue and where did you retrieve this error log?
Further, the error log shows that "You need to install the latest version of Python.". Could you confirm that you've installed [Python](https://www.python.org/).

**Comment by ysy5700** [comment]
To elaborate, I was following the instruction on https://cozie.app/docs/installation --> when I typed npm install, this error popped up. I've installed the latest Python (version: 3.10.4). The step I went through: I've installed npm, node, and also installed github desktop and then clone the cozie repository to the github desktop. > I went to cozie folder > entered cmd in the path > then type install npm > then this error message popped up.

**Comment by mariofrei** [comment]
Thanks for the update. 
Please open the CMD go to "C:\Users\윤서연\Documents\GitHub\cozie" and then type the command "npm update".
Once the update is complete there, might be a message on the second to last line that reads "npm notice Run npm install -g npm@8.6.0 to update!".
Please run that update as well.
Then you can try to resume the tutorial by typing "npm install"
Please let me know if this solves the issue or at least reduced the number of error messages.

**Comment by ysy5700** [comment]
I tried out "npm update". It didn't show  "npm notice Run npm install -g npm@8.6.0 to update!".
Then I also tried out "npm install", there was more messages came out. Please have a look below log.
----

Microsoft Windows [Version 10.0.22000.556]
(c) Microsoft Corporation. All rights reserved.

C:\Users\윤서연\Documents\GitHub\cozie>npm update
npm WARN deprecated rollup-plugin-node-resolve@5.2.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-node-resolve.
npm WARN deprecated rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
npm WARN deprecated rollup-plugin-commonjs@10.1.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-commonjs.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated core-js@2.6.12: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
npm ERR! code ECONNRESET
npm ERR! syscall read
npm ERR! errno -4077
npm ERR! network read ECONNRESET
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\윤서연\AppData\Local\npm-cache\_logs\2022-04-05T17_06_02_206Z-debug-0.log

C:\Users\윤서연\Documents\GitHub\cozie>npm install
npm WARN old lockfile
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN cleanup Failed to remove some directories [
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.acorn-QT1mNSDa',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.acorn-QT1mNSDa'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.acorn-QT1mNSDa'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.ansi-align-jB6FYJi8',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.ansi-align-jB6FYJi8'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.ansi-align-jB6FYJi8'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.define-properties-TCnDGMdU',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.define-properties-TCnDGMdU'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.define-properties-TCnDGMdU'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.convert-source-map-Jel8Xrbf',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.convert-source-map-Jel8Xrbf'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.convert-source-map-Jel8Xrbf'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.deep-extend-cljPo2N6',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.deep-extend-cljPo2N6'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.deep-extend-cljPo2N6'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.defer-to-connect-QHgkhkVw',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.defer-to-connect-QHgkhkVw'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.defer-to-connect-QHgkhkVw'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.are-we-there-yet-2UOimP31',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.are-we-there-yet-2UOimP31'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.are-we-there-yet-2UOimP31'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.mime-db-UlRk3DZr',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.mime-db-UlRk3DZr'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.mime-db-UlRk3DZr'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.make-dir-7kG1ndgY',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.make-dir-7kG1ndgY'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.make-dir-7kG1ndgY'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.form-data-MjM1PwCP',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.form-data-MjM1PwCP'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.form-data-MjM1PwCP'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.has-symbols-5z2nCgsJ',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.has-symbols-5z2nCgsJ'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.has-symbols-5z2nCgsJ'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.lru-cache-01JhAR50',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.lru-cache-01JhAR50'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.lru-cache-01JhAR50'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.replace-ext-pfVZ7ISI',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.replace-ext-pfVZ7ISI'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.replace-ext-pfVZ7ISI'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.json5-wawni35a',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.json5-wawni35a'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.json5-wawni35a'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.pofile-wmkcISZc',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.pofile-wmkcISZc'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.pofile-wmkcISZc'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.vinyl-o6aoINMW',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.vinyl-o6aoINMW'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.vinyl-o6aoINMW'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.yargs-SEbXsGgz',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.yargs-SEbXsGgz'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.yargs-SEbXsGgz'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.minimatch-MLUoG7Q9',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.minimatch-MLUoG7Q9'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.minimatch-MLUoG7Q9'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.tslib-cjgNnqOz',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.tslib-cjgNnqOz'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.tslib-cjgNnqOz'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.ansi-styles-UXp3K8Hr',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.ansi-styles-UXp3K8Hr'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.ansi-styles-UXp3K8Hr'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.inquirer-se54i14l',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.inquirer-se54i14l'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.inquirer-se54i14l'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.gulp-zip-lswT7gTz',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.gulp-zip-lswT7gTz'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.gulp-zip-lswT7gTz'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.object-keys-DATl8gqZ',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.object-keys-DATl8gqZ'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.object-keys-DATl8gqZ'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.emoji-regex-VLtOREV8',
npm WARN cleanup     [Error: EPERM: operation not permitted, rmdir 'C:\Users\윤서연\Documents\GitHub\cozie\node_modules\.emoji-regex-VLtOREV8'] {
npm WARN cleanup       errno: -4048,
npm WARN cleanup       code: 'EPERM',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'C:\\Users\\윤서연\\Documents\\GitHub\\cozie\\node_modules\\.emoji-regex-VLtOREV8'
npm WARN cleanup     }
npm WARN cleanup   ]
npm WARN cleanup ]

added 69 packages, removed 68 packages, changed 146 packages, and audited 573 packages in 21s

27 packages are looking for funding
  run `npm fund` for details

15 vulnerabilities (4 moderate, 10 high, 1 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues, run:
  npm audit fix --force

Run `npm audit` for details.

C:\Users\윤서연\Documents\GitHub\cozie>

**Comment by ysy5700** [comment]
Today I tried the same step as before on the second computer that I have. I encountered this message when I typed "npm install" on cozie folder. I had node, python, visual code installed. I tried "npm update" as well.

------

Microsoft Windows [Version 10.0.19044.1586]
(c) Microsoft Corporation. All rights reserved.

C:\Users\ysy57\Documents\GitHub\cozie>npm install
npm WARN old lockfile
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile
npm WARN deprecated rollup-plugin-node-resolve@5.2.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-node-resolve.
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated debug@3.2.6: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
npm WARN deprecated rollup-plugin-commonjs@10.1.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-commonjs.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated core-js@2.6.11: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.

added 517 packages, and audited 518 packages in 2m

9 packages are looking for funding
  run `npm fund` for details

21 vulnerabilities (5 moderate, 14 high, 2 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues, run:
  npm audit fix --force

Run `npm audit` for details.

C:\Users\ysy57\Documents\GitHub\cozie>npm audit fix --force
npm WARN using --force Recommended protections disabled.
npm WARN audit Updating @fitbit/sdk to 4.3.2,which is outside your stated dependency range.
npm WARN deprecated core-js@2.6.12: core-js@<3.4 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.

added 130 packages, removed 113 packages, changed 179 packages, and audited 535 packages in 1m

47 packages are looking for funding
  run `npm fund` for details

# npm audit report

glob-parent  <5.1.2
Severity: high
Regular expression denial of service in glob-parent - https://github.com/advisories/GHSA-ww39-953v-wcq6
No fix available
node_modules/glob-parent
  glob-stream  5.3.0 - 6.1.0
  Depends on vulnerable versions of glob-parent
  node_modules/glob-stream
    vinyl-fs  >=2.4.2
    Depends on vulnerable versions of glob-stream
    node_modules/vinyl-fs
      @fitbit/sdk  *
      Depends on vulnerable versions of vinyl-fs
      node_modules/@fitbit/sdk

4 high severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.

C:\Users\ysy57\Documents\GitHub\cozie>npm update

added 9 packages, removed 18 packages, changed 22 packages, and audited 526 packages in 4s

50 packages are looking for funding
  run `npm fund` for details

4 high severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

C:\Users\ysy57\Documents\GitHub\cozie>

**Comment by mariofrei** [comment]
Hi there,
I don't see any errors in your latest logs. The messages appear to be warnings only.
Have you tried to proceed with the next steps in the Cozie documentation?

**Comment by mariofrei** [comment]
I think this issue is solved for now. I created a [new issue](https://github.com/cozie-app/cozie/issues/112) for updating the master branch to be in line with the documentation.
